lib/ui/painting/image_generator_apng.cc

```diff
@@ -110,7 +110,15 @@ bool APNGImageGenerator::GetPixels(const SkImageInfo& info,
                     << ") of APNG due to the frame missing data (frame_info).";
     return false;
   }
-  if (frame.x_offset + frame_info.width() >
+  if (
+      // Check for unsigned integer wrapping for
+      // frame.{x|y}_offset + frame_info.{width|height}().
+      frame.x_offset >
```

Comment:
Is this also needed in `APNGImageGenerator::RenderDefaultImage`?

Reply:
Nope, as there's no offset for the default image.